src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -378,6 +378,7 @@ where
                 || lower_sql.starts_with("with")
                 || lower_sql.starts_with("describe")
                 || lower_sql.starts_with("explain")
+                || lower_sql.contains("returning")
```

Comment:
Will this be too hacky? But as we make `RETURNING` a reserved word and do not support `RETURNING` in CTE, it seems okay. 🤔 cc @BowenXiao1999

Reply:
https://github.com/risinglightdb/sqllogictest-rs/pull/141#discussion_r1062095870

Reply:
directly operating on str is always hacky. For example, I guess `insert into t values ('returning')` will be bad for this case. starts_with looks no bad. Anyway I don't quite like process str directly before use Parser, let's see whether we can remove all `is_query_sql` in future (by introduce Parser AST?, not sure).

cc @xxchan not sure whether has better ideas?

Reply:
This reminds me of https://github.com/risingwavelabs/risingwave/pull/6849#discussion_r1045645546 🤪 So we are really going to introduce parser in pgwire..

Reply:
> directly operating on str is always hacky.

+1. For sqllogictest, this is good enough. But in pgwire, we should be more careful.

Reply:
Should I revert this change?

Reply:
Yes, I think we can revert these changes that is not must-required in this pr, and add some comments to explain that it miss the ability to process `Returning` keywords.